source/about.html.slim

```diff
@@ -0,0 +1,24 @@
+---
+title: About
+---
+
+section
+  .body-inner-wrapper
+    .body-inner-wrapper__section#about
+      h2 📕 About Kiwi Ruby
+      p Kiwi Ruby is the first Ruby conference in New Zealand.
+      p Ruby is a friendly and interesting language, and the community is full of friendly and interesting people.
+      p We want to give the community the opportunity to learn and grow, and we want to showcase the talent we have right here in New Zealand.
+    .body-inner-wrapper__section#signup
+      h2 ✨ Diversity, inclusivity, and accessibility
+      p
+        ' We are committed to making Kiwi Ruby accessible and inclusive. We believe it's important that the community is diverse and can accommodate many different people.
+      p We have a <a href="http://ruby.nz/code-of-conduct">code of conduct</a> in place. We will not tolerate harassment.
+      p We have confirmed with the venue that the stage and audience are accessible to people with mobility impairments. There will be an audio loop available for people with hearing impairments.
+      p There will be childcare available onsite.
+      p If you have any accessibility needs or concerns, please get in touch with us at <a href="mailto:kiwi@ruby.nz">kiwi@ruby.nz</a>.
+    .body-inner-wrapper__section#volunteer
+      h2 🤓 Organisers
+      p Kiwi Ruby is a community event run by volunteers, overseen by <a href="https://www.ruby.nz">Ruby New Zealand</a>.
+      p The organisers are Merrin Macleod, Raquel Moss, Nahum Wild, and Philip Arndt. Between us, we have run Rails Camps, Rails Girls events, and local meetups for many years.
+      p We thought it was time for New Zealand to have a Ruby conference. It became apparent that nobody else was going to do it, so we are doing it.
```

Comment:
⚠️ I think this comes across as really snarky. What about "We thought it was time for New Zealand to have a Ruby conference, so we're making it happen"?